tests/common.rs

```diff
@@ -91,11 +91,13 @@ pub fn find_ws_url_from_output(read: impl Read + Send) -> (String, String) {
 			// does the line contain our port (we expect this specific output from substrate).
 			let sock_addr = match line.split_once("Running JSON-RPC WS server: addr=") {
 				None => return None,
-				Some((_, after)) => after.split_once(",").unwrap().0,
+				Some((_, after)) => after.split_once(',').unwrap().0,
 			};
 
 			Some(format!("ws://{}", sock_addr))
 		})
-		.expect(&format!("Could not find WebSocket address in process output:\n{}", &data));
+		.unwrap_or_else(|| {
+			panic!("Could not find WebSocket address in process output:\n{}", &data)
```

Comment:
That is a funny clippy fix - or_else ... panic: Not sure this is really better than the original.

Reply:
Because it doesn't allocate in happy path.

Reply:
It's the same basically, isn't it? But this one doesn't build the string on `Some(_)` as it is a closure?